src/components/reusable/sideDrawer/sideDrawer.scss

```diff
@@ -17,7 +17,7 @@ dialog {
   top: 0;
   right: 0;
   left: auto;
-  height: 100vh;
+  min-height: 100vh;
 
   &:modal {
     max-height: 100vh;
```

Comment:
I think there's actually a cleaner fix. Just move the `max-height: 100vh;` from `dialog:modal` to `dialog`. This happens because the `:modal` pseudo-selector goes away on close.

Reply:
Done.